### PR TITLE
Test for checking if RST is generated after connection close without read()

### PR DESCRIPTION
**Pull request by shivrai**
Test for checking whether **RST** is generated when we close the connection before **read()**ing what the client sent.

### CODE REVIEW

File: gtests/net/packetdrill/tests/bsd/tcp/close/close-unread-data-rst.pkt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+0.450 < F. 1001:1001(0) ack 1001 win 65535
+
+// TODO: ack or delayed ack ?
+0.460 > . 1001:1001(0) ack 1002 win 1026
+
+0.470 close(4) = 0
+0.480 > R. 1001:1001(0) ack 1002 win 1026
+
+// The success of the tests depends on the correctness
+// of the inbound sniffed packet on line 36.
```
Comment (tuexen) [inline gtests/net/packetdrill/tests/bsd/tcp/close/close-unread-data-rst.pkt]: Line 36 is empty... Which inbound packet are you referring to?

Comment (shivrai) [inline gtests/net/packetdrill/tests/bsd/tcp/close/close-unread-data-rst.pkt]: Fixed.

Comment (tuexen) [inline gtests/net/packetdrill/tests/bsd/tcp/close/close-unread-data-rst.pkt]: It is an outbound packet. Please fix the comment.

Comment (shivrai) [inline gtests/net/packetdrill/tests/bsd/tcp/close/close-unread-data-rst.pkt]: Done.

**Pull request merged**